templates/eap-xp1-third-party-db-s2i.json

```diff
@@ -0,0 +1,721 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP 1.0 DB driver examples",
```

Comment:
Template For JBoss EAP XP 1.0 and 3rd party DB driver

Reply:
JBoss EAP XP 1.0 and 3rd party DB driver template?

I think using 'Template .. ' at the start will be strange in the ordering of the template names in the UI and CLI, better to have what it is closer to the start?